pkgs/applications/science/electronics/openhantek6022/default.nix

```diff
@@ -0,0 +1,31 @@
+{ mkDerivation, lib, fetchFromGitHub, makeWrapper, cmake, qtbase, qttools, fftw, libusb, libglvnd }:
+
+mkDerivation rec {
+  pname = "openhantek6022";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "OpenHantek";
+    repo = "OpenHantek6022";
+    rev = version;
+    sha256 = "17b0qmz7g0nk7n7jhbh5xhs135dpj9kv41n42vvpdzwr6z5rk4qm";
+  };
+
+  nativeBuildInputs = [ cmake makeWrapper ];
+  buildInputs = [ fftw libusb libglvnd qtbase qttools ];
+
+  patches = [ ./cmake-paths.patch ./udev-rules.patch ];
+
+  postInstall = ''
+    wrapProgram $out/bin/OpenHantek \
+      --prefix QT_PLUGIN_PATH : ${qtbase}/${qtbase.qtPluginPrefix}
+  '';
+
+  meta = {
+    description = "OpenHantek6022 is a free software for Hantek and compatible (Voltcraft/Darkwire/Protek/Acetech) USB digital signal oscilloscopes.";
```

Comment:
From CONTRIBUTING.md:

`meta.description` should: Be capitalized. Not start with the package name. Not have a period at the end.

Reply:
Sorry, should have seen that